ros/angel_msgs/msg/ObjectDetection2dSet.msg

```diff
@@ -44,3 +44,10 @@ int32 descriptor_dim
 # Flattened descriptor vector. The length of this list should be equal to
 # num_detections * descriptor_dims.
 float64[] descriptors
+
+# Optional fields for Berkeley object detector.
+# Each of these vectors is expected to be of length num_detections.
+bool[] obj_obj_contact_state
+float32[] obj_obj_contact_conf
```

Comment:
@joshanderson-kw Ok, I lied a little, I was hoping to get your insight on this before I merged this.

Reply:
@hdefazio probably knows better since I get these values directly from the `decode_predictions` function in the berkeley model (https://github.com/PTG-Kitware/angel_system/blob/master/angel_system/berkeley/demo/model.py#L139). 

But it looks like there is a obj_obj_contact_class that is returned by the model to denote the state, in addition to the score.